src/extension/devtools/devtools.ts

```diff
@@ -48,94 +49,93 @@ devtools.addConnection(EXPLORER_SUBSCRIPTION_TERMINATION, () => {
   sendMessageToClient(EXPLORER_SUBSCRIPTION_TERMINATION);
 });
 
-devtools.listen(CREATE_DEVTOOLS_PANEL, ({ payload }) => {
+devtools.listen(CREATE_DEVTOOLS_PANEL, async ({ payload }) => {
   if (!isPanelCreated) {
```

Comment:
Maybe, while we're at it, inverse this and add an early `return`?
```suggestion
  if (isPanelCreated) return;
```

Reply:
Good call! Updated in [fc06c93](https://github.com/apollographql/apollo-client-devtools/pull/994/commits/fc06c930bef35564166b0fd239f2d630c56a9ccb)